docs/product/integrations/integration-platform/webhooks/issues.mdx

```diff
@@ -39,6 +39,14 @@ Sentry integrations that have subscribed to issue webhooks can receive informati
 
 ## Payload
 
+Payload includes additional information about the issue. Some important fields to pay attention to are:
+
+### status
+Some issue webhooks are getting triggered on status change of an issue. An issue status can be `resolved`, `unresolved` or `ignored` (archived).
+
+### substatus
+Substatus of an issue is giving more details about a status, specially if one status can mean multiple things. For example, an issue status can change to `unresolved` because it is now `escalating` or because a resolved issue is `regressed`. Possible substatuses are: `archived_until_escalating`, `archived_until_condition_met`, `archived_forever`, `escalating`, `ongoing`, `regressed` and `new`.
```

Comment:
```suggestion
A status can mean multiple things, so substatus gives more details about the status. For example, an issue status can change to `unresolved` because it is now `escalating` or because a resolved issue is `regressed`. Possible substatuses are: `archived_until_escalating`, `archived_until_condition_met`, `archived_forever`, `escalating`, `ongoing`, `regressed` and `new`.
```